### DISCUSSION TITLE
Why does webdriver-ts-results return 1 html file?

### DISCUSSION THREAD

**Discussion by nakrovati** [krausest/js-framework-benchmark#1358] [Q&A]
Why is 1 html file created for production?

I would like to replace webpack with vite and give the entire dist folder from webdriver-ts-results. 

In addition, it will be possible not to wait for the assembly of the production version for so long

**Comment by krausest** [comment]
Actually there are three files generated. I added code splitting some time ago to split out Plotly and some update caused a second file BoxPlotTable to appear. Code splitting happens for each `npm run results` which takes indeed some time.

There are some requirements for a refactoring:
* The resulting files must be deployable to github-pages and I want to be able to open it from the file system (without a webserver)
* All links in the files must be relative.
* If code splitting is used the resulting files should be as stable as possible (i.e. not change hashes etc. for each build)

**Reply by nakrovati** [reply]
I can promise everything, but there is a problem with viewing the output HTML file without a web server - requests to JS files are blocked by CORS rules. 

There is a workaround - vite-plugin-singlefile, but it will build everything into 1 HTML file of 2mb (react + plotly + jstat), though if you don't need it for deployment, it works quite well. I can certainly try to remove plotly from there, returning the size of the output HTML to the current one, but I'm not sure it will work. 

And so the output directory with the build can be easily deployed to GH pages, I have already checked it.
